wurst/util/Printing.wurst

```diff
@@ -1,7 +1,7 @@
 package Printing
 import NoWurst
 
-@configurable public var DEBUG_LEVEL = Loglevel.INFO
```

Comment:
Why did you change this? It's configurable. if you want to change it you can configure it inside your project. Default should stay INFO.

Reply:
Oh. I shouldn't commit it at all, sorry.